[feat] 채팅방 생성

pie2457 opened the pull request:
## Issues
- #60 

## What is this PR? 👓
채팅방 생성에 관한 PR입니다.

## Key changes 🔑
- 채팅방 생성
- 채팅방 생성 테스트

pie2457 committed:
feat #60 : 채팅방 생성

pie2457 committed:
test #60 : 채팅방 생성 테스트

23Yong commented on src/main/java/pie/tomato/tomatomarket/application/chat/ChatroomService.java:
```diff
@@ -29,4 +42,21 @@ public CustomSlice<ChatroomListResponse> findAll(Principal principal, int size,
+			.orElseThrow(() -> new NotFoundException(ErrorCode.NOT_FOUND_ITEM));
+		Member seller = getMember(item.getMember().getId());
+		
+		Chatroom saveChatroom = chatroomRepository.save(new Chatroom(LocalDateTime.now(), seller, buyer, item));
```
`저장된` 채팅방이니까 `savedChatRoom`도 좋을 것 같아요~
saveChatRoom하니까 뭔가 채팅방을 저장한다. 느낌이 나네욤

23Yong replied:
그리고 `ChatRoom` 엔티티를 생성할 때 계속해서 현재 시간을 기준으로 LocalDateTime.now()를 생성자에서 수행해도 좋을 것 같습니다.

pie2457 replied:
넵 그게 맞을거같네요 감사합니다 !

23Yong reviewed (approved):
수고하셨어요~

pie2457 committed:
refactor #60 : 리뷰사항 반영

- Chatroom, ChatroomServiceTest : 생성자에서 시간을 설정해주도록 변경
- ChatroomService : 변수명 변경 saveChatroom -> savedChatroom
